src/CommunityToolkit.Aspire.Hosting.EventStore/README.md

```diff
@@ -0,0 +1,37 @@
+# CommunityToolkit.Aspire.Hosting.EventStore library
+
+Provides extension methods and resource definitions for the .NET Aspire AppHost to support running [EventStore](https://www.eventstore.com) containers.
+
+## Getting Started
+
+### Install the package
+
+In your AppHost project, install the package using the following command:
```

Comment:
```suggestion
In your app host project, install the package using the following command:
```

Reply:
Addressed in the last commit: https://github.com/CommunityToolkit/Aspire/pull/277/commits/7deb7d5bc32aa830d8266342076c010509f75b88